os/arch/arm/src/armv7-r/arm_blocktask.c

```diff
@@ -67,6 +67,10 @@
 #include "group/group.h"
 #include "up_internal.h"
 
+#if defined(CONFIG_DEBUG_SYSTEM) && defined(CONFIG_TASK_SCHED_HISTORY)
```

Comment:
```defined(CONFIG_DEBUG_SYSTEM)``` is not needed. ```#if defined(CONFIG_TASK_SCHED_HISTORY)```  is enough. Because CONFIG_TASK_SCHED_HISTORY will be enabled only under enabled CONFIG_DEBUG_SYSTEM.
All of these kinds checking are same.

Reply:
done